bip-0003.md

```diff
@@ -21,7 +21,7 @@ address the evolving needs of the BIP process.
 
 ## Motivation
 
-BIP 2 was written in 2016.
+BIP 2 was written in 2017.
```

Comment:
I was gonna say, I’m pretty sure that it was 2016, but I see now that you are just testing the CI.

Sorry that I haven’t reviewed this yet.

Reply:
Thanks! No worries, I'm not passing the commit range correctly yet, putting it back into draft.